packages/synthetics-sdk-api/.nycrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Can you take the coverage change out of this merge request and put it in another? I also *think* that we can get the code coverage to be configured at the top level of the repo. Can you see if c8 works with monorepo?

Reply:
moved to another PR. will try to get it to work at toplevel